logstash-core/lib/logstash/webservers/puma.rb

```diff
@@ -0,0 +1,67 @@
+# encoding: utf-8
+require "logstash/webserver"
+require "puma"
+require 'puma/single'
+require 'puma/binder'
+require 'puma/configuration'
+
+class LogStash::WebServer::Puma < LogStash::WebServer::Base
+
+  extend Forwardable
+
+  attr_reader :runner, :binder, :events
+
+  def_delegator :@runner, :stats, :stats
+
+  def initialize(logger, options={})
+    super(logger, options)
+    parse_options
+
+    @runner  = nil
+    @events  = Puma::Events.strings
+    @binder  = Puma::Binder.new(@events)
+    @binder.import_from_env
+  end
+
+  def run
+    setup_signals
+    log "=== puma start: #{Time.now} ==="
+
+    @runner = Puma::Single.new(self)
```

Comment:
At my understanding, nope, we can change this by adding more thread with the ```:min_thread``` and ```:max_thread``` configuration options